packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -26,779 +31,763 @@ test.describe('modal', () => {
+                            // Assert
+                            expect(focusedElementId).toBe('default');
+                        });
                     });
                 });
             });
         });
     });
-});
 
-test.describe('`isDismissible` prop', () => {
-    test.describe('when `true`', () => {
-        test('should make the modal contain a close button', async ({ mount }) => {
-            // Arrange
-            const component = await mount(
-                PieModal,
-                {
-                    props: {
-                        isOpen: true,
-                        isDismissible: true,
+    test.describe('`isDismissible` prop', () => {
+        test.describe('when `true`', () => {
+            test('should make the modal contain a close button', async ({ mount }) => {
+                // Arrange
+                await mount(
+                    PieModal,
+                    {
+                        props: {
+                            isOpen: true,
+                            isDismissible: true,
+                        },
                     },
-                },
-            );
+                );
 
-            // Act
-            const closeButton = component.locator(closeButtonSelector);
+                // Assert
+                expect(await modalPage.isCloseButtonVisible()).toBeTruthy();
+            });
 
-            // Assert
-            await expect(closeButton).toBeVisible();
-        });
+            test('should close the modal when the close button is clicked', async ({ mount }) => {
+                // Arrange
+                await mount(
+                    PieModal,
+                    {
+                        props: {
+                            isOpen: true,
+                            isDismissible: true,
+                        },
+                    },
+                );
 
-        test('should close the modal when the close button is clicked', async ({ mount, page }) => {
-            // Arrange
-            const component = await mount(
-                PieModal,
-                {
-                    props: {
-                        isOpen: true,
-                        isDismissible: true,
+                // Act
+                await modalPage.clickCloseModal();
+
+                // Assert
+                expect(await modalPage.isModalVisible()).toBeFalsy();
+            });
+
+            test('should close the modal when the backdrop is clicked', async ({ mount }) => {
+                // Arrange
+                await mount(
+                    PieModal,
+                    {
+                        props: {
+                            isOpen: true,
+                            isDismissible: true,
+                        },
                     },
-                },
-            );
+                );
 
-            // Act
-            await page.click('[data-test-id="modal-close-button"]');
+                // Act
```

Comment:
white space

Reply:
Some of these white spaces are from when wrapped all the tests in that modal.describe block.